### ISSUE TITLE
Can't click area option

### ISSUE THREAD

**Issue by thibaultmol** [xlmnxp/blue-recorder#80]
I'm on wayland on nixos. I can record screens and windows but I can't select the option to record an area. it's not an active button I can click.

v 0.2

```
Operating System: NixOS 24.11
KDE Plasma Version: 6.1.5
KDE Frameworks Version: 6.6.0
Qt Version: 6.7.2
Kernel Version: 6.6.54 (64-bit)
Graphics Platform: Wayland
Processors: 16 × AMD Ryzen 7 7840U w/ Radeon 780M Graphics
Memory: 60.6 GiB of RAM
Graphics Processor: AMD Radeon 780M
Manufacturer: Framework
Product Name: Laptop 13 (AMD Ryzen 7040Series)
System Version: A7
```

**Comment by thibaultmol** [comment]
forgot to include screenshot 
![image](https://github.com/user-attachments/assets/589b805f-3f7a-4930-bfc3-0bfd167b77f4)

**Comment by xlmnxp** [comment]
@thibaultmol Yes, we still didn't support that feature in Wayland yet

**Comment by thibaultmol** [comment]
yeah no worries, just didn't see a Feature request for it yet, so figured it'd make one

**Comment by thibaultmol** [comment]
actually @xlmnxp could you maybe include something in a future version so there's atleast a mouseover text that appears, which says it's not available on wayland yet

**Comment by xlmnxp** [comment]
> actually @xlmnxp could you maybe include something in a future version so there's atleast a mouseover text that appears, which says it's not available on wayland yet

yes, we can do that

**Comment by codegod100** [comment]
is this a wayland limitation or an app limitation?

**Comment by xlmnxp** [comment]
@codegod100 we are working on workaround for new release #71
